charts/backingservices/charts/constellation-messaging/templates/messaging-deployment.yaml

```diff
@@ -23,6 +23,20 @@ spec:
       - name: c11n-messaging
         imagePullPolicy: {{ .Values.imagePullPolicy }}
         image: {{ .Values.image }}
+        resources:
```

Comment:
Thank you for making changes to `values.yaml`. This piece of code can be simplified as 
```
{{- if .node.resources }}
{{ toYaml .node.resources | indent 10 }}
{{- else }}
```
The idea is that there should not be a need for code changes if Clients want to pass the any other resources configuration. It will be eventually applied.

Reply:
@kishorv10 updated deployment as suggested